proxy/src/lib.rs

```diff
@@ -68,6 +69,7 @@ mod dns;
 mod drain;
 mod inbound;
 mod logging;
+mod now;
```

Comment:
TIOLI: consider just putting this in the `timeout` module and renaming it to `time`?